py-polars/polars/internals/expr/datetime.py

```diff
@@ -685,6 +685,41 @@ def second(self, fractional: bool = False) -> pli.Expr:
         │ 5.765431 │
         └──────────┘
 
+        Examples
+        --------
```

Comment:
This docstring now has two Examples sections; perhaps you meant to replace the previous section with your new examples?

Reply:
Removed now